docs/src/main/sphinx/connector/metastores.md

```diff
@@ -467,3 +464,8 @@ using Hive 3.x:
      <value>org.apache.hadoop.hive.metastore.SerDeStorageSchemaReader</value>
  </property>
 ```
+
+## Limitations
+
+Glue does not support setting comments on existing tables. You must set comments
+when [creating tables](/sql/create-table).
```

Comment:
#19073 is going to add support for table comments in Glue. I think we don't need to document this limitation.

Reply:
When is that going to land @alexjo2144 and @ebyhr ? Should we merge this PR in the meantime or close it?